openssl/src/hash.rs

```diff
@@ -46,6 +47,18 @@ impl MessageDigest {
         }
     }
 
+    pub fn from_name(name: &str) -> Option<MessageDigest> {
```

Comment:
Can you add a quick bit of documentation like what from_nid has?

Reply:
Sure.